src/integration-test/tests/legal/amountClaim/steps/amountClaims.ts

```diff
@@ -166,14 +171,14 @@ module.exports = {
     this.feeCheckForRangeTotal()
     this.verifySummaryDetails()
     this.addStatementOfTruthSignerNameAndRole()
-  },
+  }
 
   addStatementOfTruthSignerNameAndRole () {
     statementOfTruthPage.enterStatementOfTruthSignerNameAndRole()
-  },
+  }
```

Comment:
Can we please add blank between method declarations?

Reply:
done